moved/src/move_execution/tests/l2_cost.rs

```diff
@@ -0,0 +1,49 @@
+use super::*;
+
+#[test]
+fn test_treasury_charges_l1_cost_to_sender_account_on_success() {
```

Comment:
The test name still has `l1` in it instead of `l2`.

Also I think we should keep the `L1` and `L2` cost tests together, no need for separate files here.
We can call the new file `gas.rs` or `gas_cost.rs` (leaving the choice to you) and keep everything together.